examples/browser/irma-console/README.md

```diff
@@ -10,12 +10,12 @@ npm install
 npm run build
 ```
 
-Then the public directory of this example can simply be hosted using an HTTP
-server to run the example. You can for example use `serve` for this:
+To run the example, the `public` directory can simply be hosted using an HTTP
+server. You can use the [`irma` CLI tool](https://github.com/privacybydesign/irmago/releases/latest)
+to do this.
 
 ```bash
-npm install serve
-`npm bin`/serve ./public
+irma server --static-path=./public
```

Comment:
I like using the `irma server` for this, but this is not yet consistenly done everywhere: `` `npm bin`/serve`` still occurs in `irma-css/README.md`. Is this intentional?

Reply:
Yeah true, I forgot that there was a reference to an HTTP server there too. It's of course a bit of a side functionality of the IRMA server to be an HTTP server too, especially when you don't need the IRMA server at all in the example or in this case for hosting the styleguide. 

But I'll change it there too. If people have to download software anyway, it can better be the IRMA server :)

Reply:
Changed it.